feat: add docs for audience url while using auth0 (#389)
canonical/anbox-cloud-docs: howto/install-appliance/setup-oidc/auth0.md

1. From the *Settings* tab and copy the *Client ID*.

> 1. In the main navigation, select *Applications > APIs*. Copy the *API Audience* value.

Reviewer: ```suggestion
1. Select *Applications > APIs* and copy the *API Audience* value.
```
It would also be good to add a line about why we are copying this value. Something like - "This value will be used when you do xxx"

Author: > Let's try to avoid documenting every UI element, the idea is to document the goals of the user and not to document the UI as they see it. Most user interfaces are intuitive and self-explanatory.

The applications page form has several tabs and accordions. All the steps so far have been within the same page. However, this step requires the user to go to a new page. I think it’s worth keeping it this way so that users don’t look for a tab/accordion within the applications page. 

> It would also be good to add a line about why we are copying this value. Something like - "This value will be used when you configure the preseed YAML"

This is added in the next point. Should we add it to this point as well?

Reviewer: > The applications page form has several tabs and accordions. All the steps so far have been within the same page. However, this step requires the user to go to a new page. I think it’s worth keeping it this way so that users don’t look for a tab/accordion within the applications page.

I think it is counter intuitive for someone to search for Applications as an accordion when they are within the applications page. (Since we are still saying Applications > APIs)

> This is added in the next point. Should we add it to this point as well?

I would recommend providing the context here than in the next point. Users prefer to have the context when they are doing some action. So instead of asking them to do what we say mechanically, something like this may work better (this approach may be required to be applied globally):

Select *Applications > APIs*. Copy the *API Audience* value, you will need this when you configure the preseed YAML.

Now that you have the client ID, API audience URL and issuer URL, configure the preseed YAML.

Author: Makes sense, addressed the [changes](https://github.com/canonical/anbox-cloud-docs/compare/68532054fe8702fc6e39f669cb8a8850b34d6264..4a733bb31303e1382c5dd1c6e1121023690a01b3)
